src/tiles/clickable-tile.component.ts

```diff
@@ -0,0 +1,73 @@
+import {
+	Component,
+	Input,
+	OnInit
+} from "@angular/core";
+
+/**
+ * Build application's clickable tiles using this component.
+ *
+ * ## Basic usage
+ *
+ * ```html
+ * <ibm-clickable-tile>
+ * 		tile content
+ * </ibm-clickable-tile>
+ * ```
+ *
+ * @export
+ * @class ClickableTile
+ * @implements {OnInit}
+ */
+@Component({
+	selector: "ibm-clickable-tile",
+	template: `
+  <a
+    data-tile="clickable"
+    [ngClass]="{'bx--tile': true, 'bx--tile--clickable': true, 'bx--tile--is-clicked': clicked}"
+    tabindex="0"
+    (click)="handleClick($event)"
+    (keydown)="handleKeyDown($event)"
+    [href]="href"
+    [attr.aria-disabled]="disabled">
+    <ng-content></ng-content>
+  </a>`,
+	styles: ["a[aria-disabled=\"true\"] { opacity: 0.5; pointer-events: none; }"],
```

Comment:
This feels like something that should get upstreamed to carbon-components ... @asudoh thoughts?

Reply:
Also this would be more readable as a template string over multiple lines

Reply:
@cal-smith Makes sense, as long as our designers agree on “disabled clickable tile” feature. Feel free to send a GH issue our way!